sphinx_gallery/tests/test_full.py

```diff
@@ -445,7 +444,8 @@ def test_rebuild(tmpdir_factory, sphinx_app):
         new_app.build(False, [])
     status = new_app._status.getvalue()
     lines = [line for line in status.split('\n') if '0 removed' in line]
-    assert re.match('.*[0|1] added, [1-9] changed, 0 removed$.*',
+    # XXX on AppVeyor this can be 13
```

Comment:
What can be 13?

Reply:
The number of changed files in the regex below

Reply:
Ah okay. DI]id this change?

Reply:
It did in previous runs because I added `binder` to the tinybuild conf. Not sure if it's still there or not but I'm okay living with it

Reply:
Ahh I forgot binder changed. Can we change to:
`assert re.match('.*[0|1] added, [1-9][0-3]? changed, 0 removed$.*',`

Reply:
Sure, feel free to push. You could even make the `[1-9][0-3]?` only if on windows and use `[1-9]` otherwise